spec/overlap/cluster_overlap_spec.rb

```diff
@@ -6,15 +6,17 @@
 RSpec.describe Overlap::ClusterOverlap do
   include_context "with tables for holdings"
 
-  let(:spm) { build(:ht_item, enum_chron: "", billing_entity: "ucr") }
+  let(:spm) { build(:ht_item, enum_chron: "", collection_code: "PU") }
```

Comment:
Not related to the task at hand, but I kept getting failed tests due to this.
The best way to build a ht_item with a predetermined organization as `billing_entity` is to set the `collection_code` that is mapped to that organization. 

Clunky? Yes. Pretty? No.

Reply:
Yeah. It's been too annoying for too long. Looking at it.

Reply:
See e1bf96de47508d2a5547c2ec0c0be9c66b0720ea for some additions to the `ht_item` factory, and some associated tests.